bitcash/network/services.py

```diff
@@ -289,159 +117,109 @@ def get_unspent_testnet(cls, address):
         ]
 
     @classmethod
-    def get_unspent_regtest(cls, address):
-        r = requests.get(cls.REGTEST_UNSPENT_API.format(address),
-                         timeout=DEFAULT_TIMEOUT)
-        r.raise_for_status()  # pragma: no cover
-        return [
-            Unspent(currency_to_satoshi(tx['amount'], 'bch'),
-                    tx['confirmations'],
-                    r.json()['scriptPubKey'],
-                    tx['txid'],
-                    tx['vout'])
-            for tx in r.json()['utxos']
-        ]
-
-    @classmethod
-    def get_raw_transaction(cls, txid):
-        r = requests.get(cls.MAIN_RAW_API.format(
-            txid), timeout=DEFAULT_TIMEOUT)
+    def get_raw_transaction(cls, txid, network):
+        API = cls.network_endpoint(network) + cls.TX_DETAILS_PATH
+        r = requests.get(API.format(txid),
+            timeout=DEFAULT_TIMEOUT)
         r.raise_for_status()  # pragma: no cover
```

Comment:
We can drop `# pragma: no cover` comments on any `r.raise_for_status()`, I'm pretty sure.

Reply:
Rogar that. e4701d3